docs/advanced-features/compiler.md

```diff
@@ -316,6 +316,42 @@ This transform uses [handlebars](https://docs.rs/handlebars) to template the rep
 2. `member`: Has type `string`. The name of the member import.
 3. `lowerCase`, `upperCase`, `camelCase`: Helper functions to convert a string to lower, upper or camel cases.
 
+### SWC Trace profiling
```

Comment:
Is this a separate feature? Thought I remembered a different PR for this one.

Reply:
Yes, it's a separate feature and forgot to update docs then included in this PR. If it's required, can split PR into another. (feature is already in, only docs are missing)